dashboard/app/models/pd/workshop.rb

```diff
@@ -185,6 +188,16 @@ def sanitize_time_zone
     self.time_zone = time_zone.present? && ActiveSupport::TimeZone[time_zone].present? ? time_zone : nil
   end
 
+  def set_registration_link
+    reg_link = if [COURSE_CSD, COURSE_CSP, COURSE_CSA].include?(course) && local_summer?
+                 regional_partner&.link_to_partner_application.presence || "/pd/application/teacher"
+               else
```

Comment:
This else case should not be here. We are only defaulting to filling in the value for the condition in the "IF" case. Otherwise the custom reg link should be blank